reference-implementation/lib/transform-stream.js

```diff
@@ -241,7 +245,7 @@ class TransformStreamDefaultSink {
   }
 
   write(chunk) {
-    // console.log('TransformStreamDefaultSink.write()');
+    verbose('TransformStreamDefaultSink.write()');
```

Comment:
Please let's not use Class.method() for non-static methods. Use .prototype. as appropriate.

Reply:
Done.